pages/desktop/knowledge_base_new_article.py

```diff
@@ -38,6 +39,14 @@ def _page_title(self):
     _comment_box_locator = (By.ID, 'id_comment')
     _comment_submit_btn_locator = (By.CSS_SELECTOR, '.kbox-wrap .btn.btn-important')
 
+    def open(self, username, password):
+        self.selenium.get(self.canonical_url)
+        from pages.desktop.login_page import LoginPage
+        page = LoginPage(self.base_url, self.selenium).wait_for_page_to_load()
+        page.log_in(username, password)
+        self.wait_for_page_to_load()
+        return self
```

Comment:
@bobsilverberg this is an interesting use case, where opening a page requires us to go via the login page. We can't call `open` on the parent because `wait_for_page_to_load` will never return. We _could_ make that wait optional via a flag, but it's easier to just call `selenium.get` ourselves. I like that PyPOM allows for this flexibility.